contracts/user/UserManager.sol

```diff
@@ -812,23 +812,25 @@ contract UserManager is Controller, IUserManager, ReentrancyGuardUpgradeable {
     }
 
     /**
-     *  @dev Repay user's loan overdue, called only from the lending market
+     *  @dev Repay user's loan overdue, only called by UToken
      *  @param account User address
      *  @param token The asset token repaying to
      *  @param lastRepay Last repay block number
+     *  @return counter Number of frozen stakers, so we know if repay overdue loan works.
      */
     function repayLoanOverdue(
         address account,
         address token,
         uint256 lastRepay
-    ) external override whenNotPaused onlyMarketOrAdmin {
+    ) external override whenNotPaused onlyMarketOrAdmin returns (uint8 counter) {
         address[] memory stakerAddresses = getStakerAddresses(account);
-        uint256 addressesLength;
+        uint256 addressesLength = stakerAddresses.length;
         for (uint256 i = 0; i < addressesLength; i++) {
             address staker = stakerAddresses[i];
             (, , uint256 lockedStake) = getStakerAsset(account, staker);
 
             comptroller.addFrozenCoinAge(staker, token, lockedStake, lastRepay);
+            ++counter;
```

Comment:
I feel like this is just here for the purpose of the tests? I can't see if being used in UToken at all? Is this used anywhere other than the tests?

Reply:
This is also for other contracts like UToken to check if `repayOverdueLoan()` actually updates any stakers.